lib/Db/Row2Mapper.php

```diff
@@ -172,15 +172,20 @@ private function getWantedRowIds(string $userId, int $tableId, ?array $filter =
 	 * @throws InternalError
 	 */
 	public function findAll(array $showColumnIds, int $tableId, ?int $limit = null, ?int $offset = null, ?array $filter = null, ?array $sort = null, ?string $userId = null): array {
-		$this->columnMapper->preloadColumns($showColumnIds, $filter, $sort);
+		try {
+			$this->columnMapper->preloadColumns($showColumnIds, $filter, $sort);
 
-		$wantedRowIdsArray = $this->getWantedRowIds($userId, $tableId, $filter, $sort, $limit, $offset);
+			$wantedRowIdsArray = $this->getWantedRowIds($userId, $tableId, $filter, $sort, $limit, $offset);
 
-		// Get rows without SQL sorting
-		$rows = $this->getRows($wantedRowIdsArray, $showColumnIds);
+			// Get rows without SQL sorting
+			$rows = $this->getRows($wantedRowIdsArray, $showColumnIds);
 
-		// Sort rows in PHP to preserve the order from getWantedRowIds
-		return $this->sortRowsByIds($rows, $wantedRowIdsArray);
+			// Sort rows in PHP to preserve the order from getWantedRowIds
+			return $this->sortRowsByIds($rows, $wantedRowIdsArray);
+		} catch (DoesNotExistException $e) {
+			$this->logger->error($e->getMessage(), ['exception' => $e]);
+			throw new InternalError(get_class($this) . ' - ' . __FUNCTION__ . ': ' . $e->getMessage());
+		}
```

Comment:
What is the reason behind this change on the Mapper level (and the other switches to `InternalError` in this class)?